pandas-stubs/core/frame.pyi

```diff
@@ -123,10 +127,10 @@ class DataFrame:
     def append(
         self, s: Union[DataFrame, Dict[_str, Any]], ignore_index: bool = ..., sort: bool = ...
     ) -> DataFrame: ...
-    @overload
-    def apply(self, f: Callable[..., int]) -> Series: ...
-    @overload
-    def apply(self, f: Callable[..., _ListLike], axis: _AxisType = ...) -> DataFrame: ...
```

Comment:
So, can this return a series?

Reply:
Yes:

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.apply.html